.vib/opencart/cypress/cypress/integration/opencart_spec.js

```diff
@@ -0,0 +1,114 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows a user to register', () => {
+  cy.visit('/index.php?route=account/login');
+  cy.contains('Continue').click();
+  cy.fixture('users').then((user) => {
+    cy.get('#input-firstname').type(`${user.newUser.firstName}.${random}`);
+    cy.get('#input-lastname').type(`${user.newUser.lastName}.${random}`);
+    cy.get('#input-email').type(`${random}.${user.newUser.email}`);
+    cy.get('#input-telephone').type(user.newUser.phone);
+    cy.get('#input-password').type(`${user.newUser.password}.${random}`);
+    cy.get('#input-confirm').type(`${user.newUser.password}.${random}`);
+  });
+  cy.get('[type="checkbox"]').check();
+  cy.contains('Continue').click();
+  cy.contains('Your Account Has Been Created!');
```

Comment:
Something minor; to reduce a bit the flakiness, we are trying to avoid adding things like punctuation and exclamation marks to the strings we use in the `cy.contains()`.